doc/riverctl.1.scd

```diff
@@ -116,9 +116,11 @@ that tag 1 through 9 are visible.
 *enter-mode* _name_
 	Switch to given mode if it exits.
 
-*map* _mode_ _modifiers_ _key_ _command_
+*map* _mode_ [-release] _modifiers_ _key_ _command_
```

Comment:
Should this not be 
```suggestion
*map* [-release] _mode_ _modifiers_ _key_ _command_
```

Reply:
Yes. Somehow I thought in the beginning that it would be a good idea to have it there.